lib/topic_query/private_message_lists.rb

```diff
@@ -5,6 +5,15 @@ module PrivateMessageLists
     def list_private_messages(user)
       list = private_messages_for(user, :user)
       list = not_archived(list, user)
+
+      list = list.where(<<~SQL)
+        NOT (
+          topics.participant_count = 1
+          AND topics.user_id = #{user.id.to_i}
+          AND topics.moderator_posts_count = 0
+        )
```

Comment:
Sometimes you want to send yourself encrypted message as a reminder, do we want to skip this?

Reply:
Not sure about the use case but apparently we should exclude messages that you send to yourself in the list. This is something that didn't make sense to me so I removed in the the associated commit but users complained on meta so I'm adding it back ehre.